src/csharp/Microsoft.Spark/Sql/RowConstructor.cs

```diff
@@ -138,4 +67,21 @@ private StructType GetSchema()
             return schema;
         }
     }
+
+    internal sealed class RowConstructor : IObjectConstructor
```

Comment:
Can you add some high-level comment for this class. The relationship to `RowConstructorConstructor`, etc.

Reply:
added comment.